charts/cluster-api-cluster-openstack/values.yaml

```diff
@@ -72,19 +65,22 @@ openstack:
 # Modifications to this object will trigger a control plane upgrade.
 controlPlane:
   # Version of Kubernetes, should match that installed on the base images.
-  version: v1.25.2
+  version: v1.30.2
 
   # Number of control plane machines.
   replicas: 3
 
+  # Skips the kube proxy init phase in kubeadm.
+  skipKubeProxy: false
```

Comment:
You've added `registerNodes` in the schema but not added or documented it here

Reply:
Oh good spot. Yeah that was me texting the hiding of control planes. I'll remove that.